app.js

```diff
@@ -28,6 +32,9 @@ app.set('views', path.join(__dirname, './', 'views'))
 
 app.use(router)
 
-app.listen(app.get('port'), () => {
-  console.log('Server running on port:', app.get('port'))
+db.on('error', console.error.bind(console, 'connection error:'))
```

Comment:
Its basically an asynchronous way of handling an error by printing to the console